src/libraries/Microsoft.Extensions.Hosting/tests/FunctionalTests/ShutdownTests.cs

```diff
@@ -40,67 +46,67 @@ public async Task ShutdownTestWaitForShutdown()
 
         private async Task ExecuteShutdownTest(string testName, string shutdownMechanic)
         {
-            using (StartLog(out var loggerFactory))
+            var xunitTestLoggerFactory = TestLoggerBuilder.Create(builder =>
             {
-                var logger = loggerFactory.CreateLogger(testName);
-
-                // TODO refactor deployers to not depend on source code
-                // see https://github.com/dotnet/extensions/issues/1697 and https://github.com/dotnet/aspnetcore/issues/10268
+                builder.SetMinimumLevel(LogLevel.Trace);
+                builder.AddXunit(_output);
+            });
+            // TODO refactor deployers to not depend on source code
+            // see https://github.com/dotnet/extensions/issues/1697 and https://github.com/dotnet/aspnetcore/issues/10268
```

Comment:
We'll need to either do this or disable the test.  Since our tests run on helix machines without a clone of source this won't work.